perf/bin/tachyon-perf-repeat

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+function printUsage() {
+  echo "Usage: tachyon-perf-repeat <TestCase> [<Iterations>]"
+  echo -e "  The TestCase is the same as ./tachyon-perf"
+  echo -e "  The Iterations is the number of iterations to run, defaults to 10 if not specified"
+}
+
+TEST=$1
+if [ -z "${TEST}" ]; then
+  printUsage
+  exit 1
+fi
+ITERATIONS=${2:-10}
+
+bin=`cd "$( dirname "$0" )"; pwd`
+
+DEFAULT_PERF_LIBEXEC_DIR="$bin"/../libexec
+TACHYON_PERF_LIBEXEC_DIR=${TACHYON_PERF_LIBEXEC_DIR:-$DEFAULT_PERF_LIBEXEC_DIR}
+. $TACHYON_PERF_LIBEXEC_DIR/tachyon-perf-config.sh
+
+# Validate test case name
+case ${TEST} in
+SimpleRead|SimpleWrite|Metadata|SkipRead|Workflow)
```

Comment:
Oh also maybe this is not necessary. In tachyon-perf.sh, it fails at run time (but its uglier). The benefit being the cases do not need to be hardcoded.